src/modules/languages/ruby.nix

```diff
@@ -19,15 +19,20 @@ in
     packages = with pkgs; [
       cfg.package
       bundler
+      gnumake # needed for native extensions
     ];
 
     env.BUNDLE_PATH = config.env.DEVENV_STATE + "/.bundle";
 
     env.GEM_HOME = "${config.env.BUNDLE_PATH}/${cfg.package.rubyEngine}/${cfg.package.version.libDir}";
 
-    enterShell = ''
-      export GEM_PATH="$GEM_HOME/gems:$GEM_PATH"
-      export PATH="$GEM_HOME/bin:$PATH"
-    '';
+    enterShell =
```

Comment:
Let's set these via `env` instead, so they show up in `devenv info`

Reply:
AFAIK currently you cannot do env var prepends with the current system, just setting. devshell has this syntactically in it's env var implementation.